tests/func/fake_api/test_fake_api_aiohttp_serpyco.py

```diff
@@ -125,7 +126,7 @@ def get_aiohttp_serpyco_app_hapic(app):
     resp = await app.delete("/users/1")
     assert resp.status == 204
 
-
+@pytest.mark.xfail(reason='unconsistent test')
```

Comment:
Please open about the fix of this test (and add issue ref in reason)

Reply:
this one: https://github.com/algoo/hapic/issues/147 ? 
I will add ref.